app/components/Reactions/ReactionPickerFooter.css

```diff
@@ -8,4 +8,5 @@
   padding: 15px;
   background-color: var(--lego-card-color);
   border-top: 1px solid var(--border-gray);
+  border-radius: 0 0 var(--border-radius-lg) var(--border-radius-lg);
```

Comment:
There were overflow issues on the footer:

<img width="335" alt="image" src="https://user-images.githubusercontent.com/69514187/226215562-e2e71744-380b-4eda-8bba-8c089cafe8be.png">